CMakeLists.txt

```diff
@@ -285,6 +266,21 @@ drake_add_external(drake LOCAL PUBLIC CMAKE ALWAYS
     xfoil
 )
 
+# director
+drake_add_external(director PUBLIC CMAKE
+  SOURCE_SUBDIR distro/superbuild
+  CMAKE_ARGS
+    -DUSE_LCM=ON # TODO: predicate on whether we have LCM enabled?
+    -DUSE_LCMGL=ON
```

Comment:
I had changed this line to:

-DUSE_LCMGL:BOOL=${WITH_LIBBOT}

not that important but technically more correct.

Also we can remove the comment after -DUSE_LCM=ON because it is now correctly handled by the options logic (director is only available with WITH_LCM=ON)